dev-db/mariadb/mariadb-11.4.3.ebuild

```diff
@@ -281,7 +281,7 @@ src_configure() {
 	# bug #855233 (MDEV-11914, MDEV-25633) at least
 	filter-lto
 	# bug 508724 mariadb cannot use ld.gold
-	tc-ld-disable-gold
+	tc-ld-force-bfd
```

Comment:
What is the reason for this change? I miss some description anywhere, e.g. in the commit message and/or in the comment above this line.

Reply:
It's from `pkgcheck`:

```
DeprecatedEclassFunction: version 10.11.9: uses deprecated function on line 284: tc-ld-disable-gold (migrate to tc-ld-force-bfd)
```